### DISCUSSION TITLE
sensor cjmcu-811

### DISCUSSION THREAD

**Discussion by massaeg** [arendst/Tasmota#10649] [Q&A] [answered]
Dear all,
Kindly I need to know how I can use that sensor cjmcu-811 with Board ESP-32S
Thanks

**Comment by sfromis** [comment, marked as answer]
Assuming that you're already running Tasmota on the board, you should update to the tasmota32-sensors binary, configure the I2C bus and wire up the sensor. Sensor data should appear on the Tasmota home page and on MQTT.

**Reply by massaeg** [reply]
Thank you so much for your reply 💕 I will test that

**Comment by massaeg** [comment]
![Untitled](https://user-images.githubusercontent.com/64460468/105635035-99cf2a80-5e69-11eb-9f32-6fd1a7839a37.jpg)

**Reply by massaeg** [reply]
please I try to test the configure the I2C bus

**Reply by ascillato** [reply]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.
